app/src/main/java/org/coepi/android/ble/BlePreconditions.kt

```diff
@@ -24,13 +23,15 @@ class BlePreconditions(
         startPermissionsChecker.checkForPermissions(activity)
     }
 
-    fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-        bleEnabler.onActivityResult(requestCode, resultCode, data)
+    fun onActivityResult(requestCode: Int, resultCode: Int) {
```

Comment:
Can we disable the unused parameters rule? `onActivityResult` is meant to forward 1:1 the activity's `onActivityResult`, so it's better if it has the same signature.